### ISSUE TITLE
My Home: Add a card to promote webinars.

### ISSUE THREAD

**Issue by kwight** [Automattic/wp-calypso#41339]
We would like to better promote our new daily webinar series on My Home. The CTA would link to [this info page](https://wordpress.com/webinars/). Depending on design and marketing input, it could end up being a banner-style card (like our celebratory banners), or maybe an educational card (which could be either nested together in the pager card, or maybe split out below the pager card, for greater visibility.

Let's wait until design decisions are more fleshed out before starting any development.

See: pau2Xa-1bk-p2#comment4565

**Comment by mmtr** [comment]
If the experimental layout has not replaced yet the current layout on production before adding this card, let's also make sure we design a card for both layouts.

**Comment by danhauk** [comment]
Here's the first stab at this card, using the existing My Home layout but these will also work for the experimental layout (see below).

The copy still needs to be reviewed and finalized.

**Established sites**
The best way to promote this with established sites is to use the new task card pattern. It's big and clear this is the next recommended task.

<img width="872" alt="image" src="https://user-images.githubusercontent.com/448298/80257485-0608cd80-864f-11ea-8329-32784a44ef89.png">

**Pre-launch**
It's important to show this promotion even more-so in a pre-launch state. Newer customers will get a ton of value from these webinars, and we would be doing them a disservice not telling them about it early on. But we still also want to keep the focus on the site setup list.

For pre-launch, we can put a card below the list. This creates a connection between the tasks at hand and these webinars to help you through setting up your site.

This same card design could be used in the experimental layout under the Learn and grow section as well.

<img width="1148" alt="image" src="https://user-images.githubusercontent.com/448298/80257824-c68eb100-864f-11ea-9aeb-d4dbda22e8a2.png">

**Comment by Aurorum** [comment]
Sorry @danhauk - could you please share the illustration in those designs?

**Comment by Aurorum** [comment]
PR in #41460 for this issue. :)

**Comment by mmtr** [comment]
@danhauk for unlaunched sites, where should we display the card in the experimental layout? In the "Learn & Grow" section?

For reference, this is how it currently looks like:

<img src="https://user-images.githubusercontent.com/1233880/79974369-85ed3700-8499-11ea-8d80-d42e570c1f4c.png">

**Comment by kwight** [comment]
> for unlaunched sites, where should we display the card in the experimental layout? In the "Learn & Grow" section?

@mmtr I think that was [his thinking](https://github.com/Automattic/wp-calypso/issues/41339#issuecomment-619243614), yes ("This same card design could be used in the experimental layout under the Learn and grow section as well").

**Comment by kwight** [comment]
> The copy still needs to be reviewed and finalized.

@Automattic/editorial Would you be able to give some input for content please? See pau2Xa-1bk-p2#comment-4565

**Comment by michelleweber** [comment]
Is this too many words?

_Learn from the pros -- for free

Webinars with Happiness Engineers teach you to build a website, start a blog, or make money on your site. (¡También en español!)_

A little tighter:

_Learn from the pros

Free webinars with Happiness Engineers teach you to build a website, start a blog, or make money on your site._

**Comment by danhauk** [comment]
@michelleweber Thanks for these, they look pretty good. I prefer the tighter version for the sake of brevity.
